solutions/devsprint-bruno-faganello-2/SpaceApp/Utils/Extensions/View/SPViewExtension.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  SPViewExtension.swift
+//  SpaceApp
+//
+//  Created by Paolo Prodossimo Lopes on 25/04/22.
+//
+
+import SwiftUI
+
+struct SPViewExtension<V: View> {
```

Comment:
[QUESTION]
Why this?

Reply:
this is the same case for the previous question, but in this case I needed to use generic because View is a protocol

Reply:
done